server/src/index.ts

```diff
@@ -34,50 +36,59 @@ const io = new Server(socketServer, {
 
 io.on('connection', (socket: any) => {
   console.log(`[WS] User <${socket.id}> connected.`);
-  createUser(
-    "USER",
-    socket.id,
-    "AVATARURL",
-    100,
-    100,
-    "fff"
-  )
+  const defaultConnectedUser: ConnectedUser = {
+      uid: "UID",
+      socketId: socket.id,
+      displayName: "DISPLAY NAME",
+      userIcon: { 
+        foregroundImage: "FOREGROUND IMG",
+        backgroundImage: "BACKGROUND IMG"
+      },
+      location: {
+        lat: 9999,
+        lon: 9999,
+        geohash: "F"
```

Comment:
Make optional and have it so the geohash is calculated if it's not already given.